webcompat/templates/issue.html

```diff
@@ -42,6 +45,9 @@
   </main>
 {% endblock %}
 {% block extrascripts %}
+<script nonce={{nonce}}>
+  var issueData = {{ issue_data|tojson }};
```

Comment:
So uh, the lazier approach would have been to do this, and bootstrap the existing Issue model from this and call it a day. Archived issues would just have the response hard-coded, and the client would render them.

After review, if we're not happy with the mess I'm making here, I'm happy to throw this all away and submit another PR from this POV. :)